tools/Experiments/src/ViewModels/QuietBackgroundProcessesViewModel.cs

```diff
@@ -0,0 +1,218 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using DevHome.Common.Helpers;
+using Microsoft.UI.Xaml;
+using Windows.UI.Xaml;
+
+namespace DevHome.Experiments.ViewModels;
+public class QuietBackgroundProcessesViewModel : INotifyPropertyChanged
+{
+    private readonly TimeSpan _zero;
+    private readonly bool _isElevated;
+    private readonly bool _validOsVersion;
+
+    public QuietBackgroundProcessesViewModel()
+    {
+        _zero = new TimeSpan(0, 0, 0);
+
+        var osVersion = Environment.OSVersion;
+        _validOsVersion = osVersion.Version.Build >= 26024;
+        _validOsVersion = true;
+
+        if (!_validOsVersion)
+        {
+            TimeLeft = "This feature requires OS version 10.0.26024.0+";
+            return;

```

Comment:
Returning early is not ideal.  Users of this constructor might think that `QuietObjectViewModel` is in a correct state even if the os version isn't correct.  I recommend moving the OS version to methods that determine if `QuietMode` is supported.